packages/nextjs/app/builders/0x2d90C8bE0Df1BA58a66282bc4Ed03b330eBD7911/page.tsx

```diff
@@ -0,0 +1,191 @@
+                    className="text-blue-400 hover:text-blue-300 text-xs"
+                    onClick={() => copyToClipboard("0x2d90C8bE0Df1BA58a66282bc4Ed03b330eBD7911")}
+                    title="Copy full address"
+                    aria-label="Copy full Ethereum address"
+                    type="button"
+                  >
+                    [Copy]
+                  </button>
+                </div>
+
+                <p className="mt-3 text-base font-medium text-purple-300">
+                  Smart Contract Dev, In love with Cryptography
+                </p>
+
+                <p className="mt-2 text-sm text-gray-300 max-w-sm mx-auto">
+                  Exploring the boundaries of blockchain technology and passionate about building secure, efficient
+                  smart contracts.
+                </p>
+              </div>
+
+              {/* Skills chips */}
+              <div className="mt-5">
+                <div className="flex items-center gap-2 mb-3">
+                  {}
+                  {/* @ts-ignore */}
+                  <FaCode className="text-purple-400 w-4 h-4" />
+                  <h2 className="text-base font-semibold text-white">Skills</h2>
+                </div>
+                <div className="flex flex-wrap gap-2 justify-center">
+                  {skills.map(skill => (
+                    <span
+                      key={skill}
+                      className="px-3 py-1 text-sm rounded-full bg-gray-700 text-gray-200 border border-gray-600"
+                    >
+                      {skill}
+                    </span>
+                  ))}
+                </div>
+              </div>
+
+              {/* Social links */}
+              <div className="mt-6 flex justify-center gap-5">
+                <a
+                  href="https://github.com/Josetic224"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  className="social-link"
+                  aria-label="GitHub Profile"
+                >
+                  {}
+                  {/* @ts-ignore */}
+                  <FaGithub className="w-6 h-6" />
+                </a>
+                <a
+                  href="https://x.com/yoseph_Eth"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  className="social-link"
+                  aria-label="X Profile"
+                >
+                  {}
+                  {/* @ts-ignore */}
+                  <SiX className="w-5 h-5" />
+                </a>
+                <a
+                  href="https://www.linkedin.com/in/joseph-ochiagha-749853317"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  className="social-link"
+                  aria-label="LinkedIn Profile"
+                >
+                  {}
+                  {/* @ts-ignore */}
+                  <FaLinkedin className="w-6 h-6" />
+                </a>
```

Comment:
To avoid repetition, I think you can use a mapping, as you did with the `skills`.